.circleci/config.yml

```diff
@@ -7,24 +7,24 @@ jobs:
     steps:
       - checkout
       - run:
-          command: npm ci
+          command: yarn install --frozen-lockfile
       - run:
-          command: npm run build
+          command: yarn build
   lint:
     docker:
       - image: circleci/node:14.17.6
     steps:
       - checkout
       - restore_cache:
-          key: package-cache-{{ checksum "package-lock.json" }}
+          key: package-cache-{{ checksum "yarn.lock" }}
       - run:
-          command: npm install
+          command: yarn
```

Comment:
Oop guess I forgot to mark all the instances of `yarn` but should these also be `yarn install --frozen-lockfile`? I see it was `nom install` instead of `npm ci` and wasn't sure if that was intentional.